src/Jackett.Common/Definitions/bibliotik.yml

```diff
@@ -0,0 +1,146 @@
+---
+  site: bibliotik
+  name: Bibliotik
+  description: "Bibliotik is a private site for eBooks and audiobooks"
+  language: en-US
+  type: private
+  encoding: UTF-8
+  links:
+    - https://bibliotik.me/
+
+  caps:
+    categorymappings:
+      - { id: 1, cat: PC, desc: "Applications" }
+      - { id: 3, cat: Audio/Audiobook, desc: "Audiobooks" }
+      - { id: 4, cat: Books/Comics, desc: "Comics" }
+      - { id: 5, cat: Books/Ebook, desc: "eBooks" }
+      - { id: 7, cat: Books/Magazines, desc: "Magazines" }
+
+    modes:
+      search: [search]
+
+  settings:
+    - name: username
+      type: text
+      label: Username
+    - name: password
+      type: password
+      label: Password
+    - name: orderby
+      type: select
+      label: Sort requested from site
+      default: "relevance"
+      options:
+        "relevance": "relevance"
+        "year": "year"
+        "size": "size"
+        "duration": "duration"
+        "added": "added"
+        "leechers": "leechers"
+        "seeders": "seeders"
+        "snatches": "snatches"
+        "comments": "comments"
+        "files": "files"
+    - name: order
+      type: select
+      label: Order requested from site
+      default: "desc"
+      options:
+        "desc": "desc"
+        "asc": "asc"
+
+  login:
+    path: /
+    method: form
+    form: form#loginform
+    submitpath: /
+    cookies: ["__cfduid", "id"]
+    inputs:
+      action: login
+      username: "{{ .Config.username }}"
+      password: "{{ .Config.password }}"
+      keeploggedin: 1
+    error:
+      - selector: center:first-of-type
+    test:
+      path: torrents/
+      selector: div#pre_header
+
+  search:
+    paths:
+      - path: torrents/
+    inputs:
+      search: "{{ .Keywords }}"
+      $raw: "{{ range .Categories }}cat[]={{.}}&{{end}}"
+      orderby: "{{ .Config.orderby }}"
+      order: "{{ .Config.order }}"
+
+    rows:
+      selector: table#torrents_table > tbody > tr:has(.title)
+
+    fields:
+      category:
+        selector: td:first-child
+        case:
+          div[title="Applications"]: 1
+          div[title="Audiobooks"]: 3
+          div[title="Comics"]: 4
+          div[title="Ebooks"]: 5
+          div[title="Magazines"]: 7
+          "*": 0
```

Comment:
the case default clause you have set up sets category to 0, but you have not defined a categorymapping for id:0
I suspect this will generate an error anytime this selector returns 0.
but maybe that was on purpose, so that we get notified of a new category on bibliotik?
but don't go adding a categorymapping for id:0 if bibliotik does not have one, because that could cause an error when users do a search with filter cat0, so that could become a problem too.
does bibliotik not have an **Other** cat? because that would be the ideal default cat to use.

Reply:
No there is no **Other** cat, oddly enough. I believe you're right that we should just omit it, I was just following patterns I saw elsewhere, but Bibliotik seems like an odd case in only having a few explicit categories.